config/webpack/webpack.common.js

```diff
@@ -212,6 +212,26 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
             'process/browser': require.resolve('process/browser'),
         },
     },
+    optimization: {
+        runtimeChunk: 'single',
+        splitChunks: {
+            cacheGroups: {
+                // Extract all 3rd party dependencies (~75% of App) to separate js file
+                // This gives a more efficient caching - 3rd party deps don't change as often as main source
+                // When dependencies don't change webpack would produce the same js file (and content hash)
+                // After App update end users would download just the main source and resolve the rest from cache
+                // When dependencies do change cache is invalidated and users download everything - same as before
+                vendor: {
+                    test: /[\\/]node_modules[\\/]/,
```

Comment:
This is really awesome because the build is so much faster now, but one downside I found it was a pain for developing locally with `react-native-onyx` and that I have to clear the webpack cache anytime I wanted to test a new change. What about adding an exclusion just for `react-native-onyx`?

Reply:
I don't think the part of the config here (vendor chunk) would make a difference about that 

Does it work if we revert this vendors chunk change? 
If it does we can keep it for prod but remove it from the dev config 

But I think the problem is from file system cache config - webpack assumes node modules does not change while running and just uses cached values. Perhaps there's a way to tell it certain modules might change